cmd/tempo-cli/cmd-rewrite-blocks.go

```diff
@@ -131,7 +153,13 @@ func rewriteBlock(ctx context.Context, r backend.Reader, w backend.Writer, meta
 
 		// hook to drop the trace
 		DropObject: func(id common.ID) bool {
-			return bytes.Equal(id, traceID)
+			// Should it be replaced with binary search, trie or other ones?
+			for _, tid := range traceIDs {
+				if bytes.Equal(id, tid) {
```

Comment:
I think this if can be in lineed in the return like `return bytes.Equal(id, tid)`

Reply:
The loop checks if `traceIDs` contains `id`, while `return bytes.Equal(id, tid)` would only compare `id` to a single value, missing the iteration over all elements.